pygac/klm_reader.py

```diff
@@ -33,6 +33,11 @@
 
 import datetime
 import logging
+try:
+    from enum import IntFlag
+except ImportError:
+    # python version < 3.6, use a simple object without nice representation
+    IntFlag = object
```

Comment:
Can we decide that pygac is python >= 3.6 only? It would make a lot simpler

Reply:
Absolutely, I would be one of the first to vote for it.
There are already a lot of special python 2 hacks and test cases that we skip, so it would totally make sense.
I don't know all PyGAC users, but nowadays it is easy to set up a python >= 3.6 environment using anaconda, so I guess no one will complain.

Reply:
@sfinkens @abhaydd any opinion on this?

Reply:
I think that's fine. But let's do this in a separate PR ;)